apps/vr-tests-web-components/src/stories/badge/badge.stories.tsx

```diff
@@ -0,0 +1,74 @@
+import * as React from 'react';
```

Comment:
## [🕵 fluentui-web-components-v3 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28794&commitId=086e5cfafa0699a4b60690abfddceff748cc5e0e&env=prod&clientType=FLUENTUI)


  <details><summary><b>Badge</b> 10 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Badge.Appearance - Dark Mode.normal.chromium.png|0| Added|
|Badge.Appearance.normal.chromium.png|0| Added|
|Badge.Color - Dark Mode.normal.chromium.png|0| Added|
|Badge.Color.normal.chromium.png|0| Added|
|Badge.Default - Dark Mode.normal.chromium.png|0| Added|
|Badge.Default.normal.chromium.png|0| Added|
|Badge.Shape - Dark Mode.normal.chromium.png|0| Added|
|Badge.Shape.normal.chromium.png|0| Added|
|Badge.Size - Dark Mode.normal.chromium.png|0| Added|
|Badge.Size.normal.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>